monai/transforms/utils.py

```diff
@@ -447,7 +447,10 @@ def correct_crop_centers(
     spatial_size = fall_back_tuple(spatial_size, default=label_spatial_shape)
     if any(np.subtract(label_spatial_shape, spatial_size) < 0):
         if not allow_smaller:
-            raise ValueError("The size of the proposed random crop ROI is larger than the image size.")
+            raise ValueError(
+                "The size of the proposed random crop ROI is larger than the image size, "
+                f"got {label_spatial_shape} and {spatial_size} respectively."
```

Comment:
```suggestion
                f"got {spatial_size} and {label_spatial_shape} respectively."
```
Maybe it could be reversed this way to correspond to “proposed random crop ROI” and “image size”?